multipledispatch added as dev dependency but needed in production

# Brief Description

<!-- Please provide a brief description of your bug. Do NOT paste the stack trace here. -->
In this commit (https://github.com/ericmjl/pyjanitor/commit/331f4a858055079d1504c36c543e72c6c19bca6b) the library is now used in a function but it was just added to dev dependencies. I think this may have happened before. It fails for my app's CICD which imports PyJanitor. Does PyJanitor have a CICD process that doesn't import dev dependencies?


# Error Messages

<!-- If you get an error message, please paste it between the backticks here. -->

```
/usr/local/lib/python3.8/site-packages/janitor/functions.py:28: in <module>
    from multipledispatch import dispatch
E   ModuleNotFoundError: No module named 'multipledispatch'
```

--- a/.requirements/base.in
+++ b/.requirements/base.in
@@ -5,3 +5,4 @@ natsort
 # seaborn
 pandas_flavor
 sklearn
+multipledispatch

